app/models/pbs/event/participation.rb

```diff
@@ -12,6 +12,7 @@ module Pbs::Event::Participation
   included do
     validates :bsv_days, numericality: { greater_than_or_equal_to: 0, allow_blank: true }
     validate :assert_bsv_days_precision
+    after_create :send_black_list_mail
```

Comment:
Good advice, I thought it's not really recommended because I did not found any Implementation like that in the existing code. But I will certainly replace it with my current code :)